src/mocks/handlers.ts

```diff
@@ -142,7 +143,15 @@ export const handlers = [
     }),
     rest.get(API_PATH + "/pages/pods/:tag", (req, res, ctx) => {
         const {tag} = req.params;
-        const podsFilteredByTag = produce(mockNewsPods, pods => {
+        let mockPods;
```

Comment:
You can simplify this by writing it like this:
```typescript
const mockPods = tag === "promo" ? mockPromoPods : mockNewsPods;
```

If you've used a switch to make it easier to add new tags in the future, then it's better not to do that, because that may never happen and your code is harder to read. It would be a bit of a case of premature optimisation.

Reply:
Yeah I was trying to keep it so that it would be easily useable for further tags, but I'll simplify as suggested!

Reply:
I understand. In these cases, unless you can guarantee that changes are definitely going to happen imminently (like in the current or next sprint), it's better to just keep it simple. Because otherwise what ends up happening in the vast majority of cases is that the use case you were foreseeing never actually happens, and the result is that the code is harder to read than it needed to be.